src/System.IO.FileSystem/tests/Directory/ReparsePoints_MountVolume.cs

```diff
@@ -0,0 +1,435 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+/**
+This testcase attempts to checks GetDirectories/GetFiles with the following ReparsePoint implementations
+ - Mount Volumes
+**/
+using System;
+using System.IO;
+using System.Text;
+using System.Collections.Generic;
+using System.Diagnostics;
+using Xunit;
+
+public class Directory_ReparsePoints_MountVolume
+{
+    private delegate void ExceptionCode();
+    private static bool s_pass = true;
+
+    [Fact]
+    [ActiveIssue(0)]
```

Comment:
Need something here other than just ActiveIssue(0).  Either file an issue or delete the test.